l10n_ch_pain_credit_transfer/__manifest__.py

```diff
@@ -14,5 +14,5 @@
         "l10n_ch_base_bank",
         "account_banking_sepa_credit_transfer",
     ],
-    'installable': True,
```

Comment:
Please include the modifications done by linters in a separated commit as explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#technical-method-to-migrate-a-module-from-130-to-140-branch